### ISSUE TITLE
More expect options

### ISSUE THREAD

**Issue by vuquangtam** [simenheg/restclient-test.el#1]
Can you support to add the expect response content ( compare response content with some string or regex match ) in the future. Thanks

**Comment by simenheg** [comment]
Yes, that would be nice. I also think it could be useful to be able to define general functions for checking the response data. I'll see if I can find the time to implement something like this soon!
